src/models/blueprint.js

```diff
@@ -93,6 +98,46 @@ export default class Blueprint {
     }
   }
 
+  static loadAll(options = {}) {
```

Comment:
I get a little antsy with long files.  I think we could separate the "find, load and instantiate" parts from the specific blueprint objects.  `blueprintLoader.js` maybe?

Reply:
Works for me. Just wanted a quick fix to get the cli piece working in the first instance.

Reply:
Splitting/expanding functionality in the Blueprint may warrant a separate PR?

Reply:
Yes, that is the best thing to do.  I just started commenting on things tagential to many different issues and it all got a but muddled.